src/utils/MastodonUtils.C

```diff
@@ -10,6 +10,9 @@
 #include "tinydir.h"
 #include "MooseUtils.h"
 
+#include <iostream>
```

Comment:
Why do we need iostream and fstream here?

Reply:
Yeah, they're not required.